frontend/src/components/FindModal/FindModal.tsx

```diff
@@ -79,7 +79,6 @@ const FindModal = <T extends unknown>(
                   name={props.name}
                   data={props.query.data}
                   isLoading={props.query.isLoading}
-                  isFetching={props.query.isFetching}
                   error={props.query.error}
                   columns={props.columns}
                   onRowClick={handleRowClick}
```

Comment:
Refactor this file and use destructuring for all the props. I know it's not your main goal for this issue, but the component would much cleaner 😎